src/crepe/api/Texture.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "Asset.h"
+#include <SDL2/SDL_render.h>
```

Comment:
Deze include mag niet omdat deze header niet langer beschikbaar is zodra alleen crepe geïnstalleerd is